docs/docs/user/getting-started/components.md

```diff
@@ -0,0 +1,529 @@
+# Components
+
+This section explains the different components of KPOps, their usage and configuration.
+To learn more about their hierarchy, visit the
+[architecture](/kpops/docs/docs/developer/architecture/component-inheritance.md)
+page.
+
+## KubernetesApp
+
+### Usage
+
+Can be used to deploy any app in Kubernetes using helm, for example, a REST service that serves Kafka data.
+
+### Configuration
+
+??? "`pipeline.yaml`"
+
+    ```yaml
+      # Base Kubernetes App
+    - type: kubernetes-app
+      name: kubernetes-app # required
+      namespace: namespace # required
+      # `app` contains application-specific settings, hence it does not have a rigid
+      # structure. The fields below are just an example.
+      app: # required
+        key1:
+          key4: value3
+        key2: value1
+        key3: value2
+      # Topic(s) from which the component will read input
+      from:
+        topics: # required
+          ${pipeline_name}-input-topic:
+            type: input # required
+            role: topic-role
+      # Topic(s) into which the component will write output
+      to:
+        topics: # required
+          ${pipeline_name}-output-topic:
+            type: output # required
+            role: topic-role
+            keySchema: key-schema
+            valueSchema: value-schema
+            partitions_count: 1
+            replication_factor: 1
+            configs: 
+              config1: config1
+        models: 
+          model: model
+      # Pipeline prefix that will prefix every component name. If you wish to not 
+      # have any prefix you can specify an empty string.
+      prefix: ${pipeline_name}-
+      # Helm repository configuration
+      repoConfig:
+        repositoryName: my-repo # required
+        url: https://bakdata.github.io/ # required
+        repoAuthFlags: 
+          username: user
+          password: pass
+          ca_file: /home/user/path/to/ca-file
+          insecure_skip_tls_verify: false
+      version: 1.0.0
+    ```
+
+### deploy
+### destory
+### reset
+### clean
+
+## KafkaApp
+
+_KafkaApp_ inherits from [_KubernetesApp_](#kubernetesapp).
+
+### Usage
+
+Inherits from KubernetesApp. Often used in `defaults.yaml` Not usually used to deploy Kafka applications as they almost could be defined as either a _StreamsApp_ or a _Producer_.
```

Comment:
Please make it clear that this deploys a streams-bootstrap component which can be (as you wrote) StreamsApp or Producer

Reply:
Done